book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md

```diff
@@ -0,0 +1,57 @@
+# Proof of Concept: A Cross-Layer Transferal System
+
+---
+
+We have successfully initiated our Layer 2 - Layer 2 asset transferal system (Mosaic) with a Proof of Concept (PoC) linking Polygon, Fantom, Arbitrum, the Avalanche C-Chain, Moonriver, and the Ethereum mainnet (layer 1). 
+
+Polygon, Arbitrum, and Avalanche are leading L2/scalability solutions for Ethereum, with all being used by many of the largest DeFi protocols. For instance, [nearly 100 protocols](https://www.block123.com/en/feature/polygon-matic-network-list/) have built on Polygon including MakerDAO's [DAI](https://makerdao.com/en/) (with [MakerDAO](https://makerdao.com/)being the second largest DeFi protocol at the time of writing, having over $13.5 billion in TVL, or total value locked), [Chainlink](https://chain.link/), and [Aavegotchi](https://aavegotchi.com/). Arbitrum has been incorporated into major DeFi protocols like [Bancor](https://bancor.network/) (currently the 13th largest DeFi project in terms of TVL), [Augur](https://augur.net/) and others. Avalanche similarly has an impressive [343 protocols](https://www.avax-projects.com/) built along it at the time of writing.
```

Comment:
The `nearly 100 protocols` link doesn't DNS resolve for me, can someone check?

Reply:
There is an old version in the web archive. But it looks like this was an ever evolving page, showing different Polygon projects. So I'm not sure if it'd make sense, to replace this link with the archive one.
https://web.archive.org/web/20210725112237/https://www.block123.com/en/feature/polygon-matic-network-list/

Reply:
replaced link with archive version